src/models/instance.ts

```diff
@@ -31,6 +31,30 @@ export default class Instance {
         this.data = _.pick(data, ["id", "name", "url", "username", "password", "description"]);
     }
 
+    public get id(): string {
+        return this.data.id;
+    }
+
+    public get name(): string {
+        return this.data.name;
+    }
+
+    public get url(): string {
+        return this.data.url;
+    }
+
+    public get username(): string {
+        return this.data.username;
+    }
+
+    public get password(): string {
+        return this.data.password;
+    }
+
+    public get description(): string {
+        return this.data.description ? this.data.description : "";
```

Comment:
[ultra-minor] [Note: not for this refactor PR] `x ? x : y` -> `x || y`.

Reply:
Yeah, I tend to forget this exists! Thanks @tokland